docs/documentation/server_admin/topics/clients/oidc/con-audience.adoc

```diff
@@ -103,16 +103,16 @@ image:images/audience_mapper.png[]
 On the confidential client:
 +
 . Click the _Client Scopes_ tab.
-. Assign *good-service* as an optional (or default) client scope. 
+. Assign *good-service* as an optional (or default) client scope.
 +
 See <<_client_scopes_linking, Client Scopes Linking section>> for more details.
 
 * You can optionally <<_client_scopes_evaluate, Evaluate Client Scopes>> and generate an example access token. *good-service* will be added to the audience of the generated access token if *good-service* is included in the _scope_ parameter, when you assigned it as an optional client scope.
 
-* In your confidential client application, ensure that the _scope_ parameter is used. The value *good-service* must be included when you want to issue the token for accessing *good-service*. 
+* In your confidential client application, ensure that the _scope_ parameter is used. The value *good-service* must be included when you want to issue the token for accessing *good-service*.
 +
 See:
 +
-** link:{adapterguide_link}#_javascript_adapter[javascript adapter section] if your application uses the javascript adapter.
+** *Keycloak JavaScript adapter* in the link:https://www.keycloak.org/guides#securing-apps[securing apps] section if your application uses the javascript adapter.
```

Comment:
Same like my other comment (can be hopefully attribute instead of hardcoded link).